.github/workflows/ci.yaml

```diff
@@ -71,6 +71,9 @@ jobs:
             - name: Run Integ Tests
               run: npm run integ:local:${{ matrix.browser }}:headless
 
-            - name: Run IE Integ Tests
+            - name: Run IE,Edge,Firefox Integ Tests
               if: matrix.os == 'windows-latest'
-              run: npm run integ:local:ie
+              run: |
+                  npm run integ:local:ie
+                  npm run integ:local:edge
+                  npm run integ:local:firefox
```

Comment:
Firefox can go under the `Run Integ Tests` step. The [Ubuntu image](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md) has Firefox installed.

Edge also runs on Linux, but it's not installed by default, so I'm ok leaving it as Windows-only.